test/extended/testdata/bindata.go

```diff
@@ -5347,6 +5348,70 @@ func testExtendedTestdataBuildsValuefromTestSecretYaml() (*asset, error) {
 	return a, nil
 }
 
+var _testExtendedTestdataCliPodWithTwoContainersYaml = []byte(`kind: Pod
+apiVersion: v1
+metadata:
+  name: doublecontainers
+  creationTimestamp: 
+  labels:
+    name: hello-openshift
+spec:
+  containers:
+  - name: hello-openshift
+    image: aosqe/hello-openshift
+    ports:
+    - containerPort: 8080
+      protocol: TCP
+    resources:
+      limits:
+        memory: 256Mi
+    volumeMounts:
+    - name: tmp
+      mountPath: "/tmp"
+    terminationMessagePath: "/dev/termination-log"
+    imagePullPolicy: IfNotPresent
+    capabilities: {}
+    securityContext:
+      capabilities: {}
+      privileged: false
```

Comment:
Above are non-used fields for this PR, should be trimmed to make the YAML least.